Core/Class.cs

```diff
@@ -0,0 +1,28 @@
+using System;
+
+namespace PCSharpGen.Core
+{
+    public class Class
+    {
+        // Maybe should be an enum?
+        public string Name { get; set; }
+
+        public int HitDie { get; set; }
+
+        public int SkillRanks { get; set; }
+
+        public float BaseAttackBonus { get; set; }
```

Comment:
Similar to the save progression comment below, I think BAB should probably be a lambda.